raiden-cli/src/config.json

```
@@ -4,6 +4,20 @@
   "autoSettle": true,
   "autoUDCWithdraw": false,
   "caps": {
-    "Receive": 1
+    "Receive": 1,
+    "Mediate": 1,
+  },
+  "mediationFees": {
+    "0x0000000000000000000000000000000000000000": {
+      "flat": 0, 
+      "prop": 4000,
+      "inbalance": 3000,
```

Not sure if my `imbalance`'s spelling is correct, but that's how it's currently in code:
```suggestion
      "proportional": 4000,
      "imbalance": 3000,
```

---

Also, for the default config, I'd add the explicit `"cap": true`, even if the default is hardcoded.

---

Fu** I'm stupid. Sorry. :see_no_evil:

---

✔️